README.md

```diff
@@ -21,13 +21,35 @@ Inspired by the sheer number of other 'awesome' lists out there.
 
 ## Reference material
 
+ - Hedler, Marko, Manuel Montero Pineda, and Nico Kutscherauer. “[Schematron: Effiziente Business Rules für
+   XML-Dokumente.](https://schematron.info)” Heidelberg: dpunkt, 2011.
+ - Jelliffe, Rick. “[The Schematron Assertion Language
+   1.6](https://web.archive.org/web/20061230150144/http://xml.ascc.net:80/resource/schematron/Schematron2000.html)”
+   Online, October 1, 2002.
+ - Maus, David. “[Ex-Post Rule Match Selection: A Novel Approach to XSLT-Based Schematron
+   Validation.](https://archive.xmlprague.cz/2019/files/xmlprague-2019-proceedings.pdf#page=79)” In XML Prague 2019
+   Conference Proceedings, 57–65. Prague, Czech Republic, 2019.
+ - Maus, David. “[An Introduction to Schematron and Schematron QuickFix.](https://doi.org/10.5281/zenodo.3429706)”
+   Presented at the TEI Conference and Member’s Meeting 2019 (TEI 2019), Graz, Austria, 16–22 September 2019, Graz,
+   September 16, 2019.
+ - Maus, David. “[Schematron Report Customization.](https://www.youtube.com/watch?v=V5M_mRLoy8s)” Webinar presented at
+   the Markup UK Solidarity Edition, June 10, 2020.
+ - Maus, David. “[What’s in a Schematron?](https://markupuk.org/webhelp/index.html#ar02.html)” In Markup UK 2021
```

Comment:
The "What's is a Schematron?" line will go out of date when the current proceedings are archived to make way for Markup UK 2022's proceedings.